en/orm/retrieving-data-and-resultsets.rst

```diff
@@ -497,8 +497,9 @@ load fields off of contained associations, you can use ``autoFields()``::
 
     // Select id & title from articles, but all fields off of Users.
     $query->select(['id', 'title'])
-        ->contain(['Users'])
-        ->autoFields(true);
+        ->contain(['Users' => function($q) {
+			return $q->autoFields(true);
+		          }]);
```

Comment:
I could see autoFields() being necessary for associations that are not loaded with joins. HasMany and BelongsToMany use secondary queries, so autoFields() makes sense there. But in a joinable assocations (belongsTo, hasOne) that aspect of the query is not copied over.